tests/Assetic/Test/Asset/HttpAssetTest.php

```diff
@@ -53,9 +53,9 @@ public function testInvalidUrl()
 
     public function testSourceMetadata()
     {
-        $asset = new HttpAsset(self::JQUERY);
-        $this->assertEquals('http://ajax.googleapis.com', $asset->getSourceRoot(), '->__construct() set the source root');
-        $this->assertEquals('ajax/libs/jquery/1.6.1/jquery.min.js', $asset->getSourcePath(), '->__construct() set the source path');
-        $this->assertEquals('http://ajax.googleapis.com/ajax/libs/jquery/1.6.1', $asset->getSourceDirectory(), '->__construct() sets the source directory');
+        $asset = new HttpAsset(self::ASSET_URL);
+        $this->assertEquals('https://maxcdn.bootstrapcdn.com', $asset->getSourceRoot(), '->__construct() set the source root');
```

Comment:
I recommend doing `parse_url` on `self::ASSET_URL` and then using what it returns in assertion. This way less duplication of url fragments in tests.